src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -42,17 +42,20 @@ class TextWrapper
 {
     private $app;
     private $env;
+    private $config_path;
+    private $parser;
     private $exit_code;
 
     /**
-     * @param  Phinx\Console\PhinxApplication $app
-     * @param  String                         $default_env  environment fallback, defaults to "development"
-     * @return void
+     * @param PhinxApplication $app
+     * @param array $options
      */
-    public function __construct(PhinxApplication $app, $default_env = 'development')
+    public function __construct(PhinxApplication $app, Array $options = array())
     {
         $this->app = $app;
-        $this->env = $default_env;
+        $this->env = (isset($options['environment']) ? $options['environment'] : 'development');
```

Comment:
It's better to accept an array, and then have a `getOption` method, like so:

```php
protected function getOption($name)
{
    if (!isset($this->options[$name])) {
        return null;
    }
    return $this->options[$name];
}
```

That way an option can be added in the future without having to modify code in two places.